### PR TITLE
TESTS-52: Changing testing url from dev to staging

### PR DESCRIPTION
**Pull request by Ravineesh**
Changing testing url from dev to staging for time being.

### CODE REVIEW

**Comment by Hassan754** [comment]
Why do you need to change the testing url in the first case

**Comment by Ravineesh** [comment]
> Why do you need to change the testing url in the first case



> Why do you need to change the testing url in the first case

It is required for time being to pass one of the test cases related to collaboration where clients will use the trade using trading script.

**Review by faizan2590 — APPROVED** [review]
LGTM

**Review by Hassan754 — APPROVED** [review]

File: tests/e2e_test_auto_offer_on_device_rest.py [hunk 1]
```diff
@@ -18,8 +18,8 @@
 args = parser.parse_args()
 connected_devices = []
 
-domain_name = 'https://d3aweb-dev.gridsingularity.com'
-websockets_domain_name = 'wss://d3aweb-dev.gridsingularity.com/external-ws'
+domain_name = 'https://d3aweb-staging.gridsingularity.com'
+websockets_domain_name = 'wss://d3aweb-staging.gridsingularity.com/external-ws'
```
Comment (hannesdiedrich) [inline tests/e2e_test_auto_offer_on_device_rest.py:22]: @Ravineesh Sorry, just saw that. I would have suggested to add these to variables as a argument to the script to easily be able to switch between domains from the d3a-e2e-ui-test side.

Comment (Ravineesh) [reply tests/e2e_test_auto_offer_on_device_rest.py:22]: Ok thanks. I'll change it when I'll switch it back to dev url.

**Pull request merged**